doc/api-examples/rust/setup.md

```diff
@@ -0,0 +1,36 @@
+<!-- markdownlint-disable-file MD013 MD036 MD041 -->
+**Setup sample project**
+
+The Rust snippets require at least a simple project setup. Create it via:
+
+```bash
+# initialize the project
+cargo new bluechi
+cd bluechi
+```
+
+**Installing dependencies**
+
+The Rust snippets require the [dbus](https://docs.rs/dbus/latest/dbus/index.html) create. Navigate into the `bluechi` directory and add it as a dependency via:
+
+```bash
+cargo add dbus
```

Comment:
Wouldn't it be better to use

```
dnf install rust-dbus-devel
```

Reply:
responded https://github.com/containers/bluechi/pull/563#discussion_r1325752617